src/app/core/services/vault/vault.service.ts

```diff
@@ -46,11 +49,31 @@ export class VaultService {
 
       chrome.storage.sync.set({
         fctAddresses: [],
-        ecAddresses: []
+        ecAddresses: [],
+        fctAddressesRequestWhitelistedDomains: [],
+        ecAddressesRequestWhitelistedDomains: []
       })
     });
   }
 
+  updateStorageState() {
+    const state = this.localStorageStore.getState();
+    if (state.version === this.lastStableLocalStorageVersion) {
+      const newState = Object.assign({}, state, {
+        version: environment.localStorageVersion,
+        fctAddressesRequestWhitelistedDomains: JSON.stringify([]),
+        ecAddressesRequestWhitelistedDomains: JSON.stringify([])
+      });
+
+      this.localStorageStore.putState(newState);
+
+      chrome.storage.sync.set({
+        fctAddressesRequestWhitelistedDomains: [],
+        ecAddressesRequestWhitelistedDomains: []
+      });
+    }
+  }
```

Comment:
When the user opens the extension if the localStorageStore version is '1.0', the localStorageStore state will be updated